package.json

```diff
@@ -76,7 +76,7 @@
     "prettier": "^1.19.1",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "^3.4.1",
+    "react-scripts": "4.0.0",
```

Comment:
Why not `^4.0.0`? Is there a particular reason?

Reply:
I wanted to see which package dependencies would get updated and bumped it according to the docs on migration from 3.4.4 -> 4.0.0

This is a `wip` and wanted to take some time to discuss options in our meeting tomorrow.

Reply:
Works for me :)